recipes/earcut/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class EarcutConan(ConanFile):
+    name = "earcut"
+    description = "A C++ port of earcut.js, a fast, header-only polygon triangulation library."
+    homepage = "https://github.com/mapbox/earcut.hpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "ISC"
+    topics = ("algorithm", "cpp", "geometry", "rendering", "triangulation",
+              "polygon", "header-only", "tessellation", "earcut")
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _minimum_compilers_version(self):
+        # References:
+        # * https://github.com/mapbox/earcut.hpp#dependencies
+        # * https://en.cppreference.com/w/cpp/compiler_support/11
+        # * https://en.wikipedia.org/wiki/Xcode#Toolchain_versions
+        return {
+            "apple-clang": "5.1",
+            "clang": "3.4",
+            "gcc": "4.9",
+            "intel": "15",
+            "sun-cc": "5.14",
+            "Visual Studio": "12"
+        }
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```
https://blog.conan.io/2020/12/14/New-conan-release-1-32.html